css-grid-layout/esra/styles.css

```diff
@@ -10,18 +12,41 @@ body {
 
 .header {
   background-color: #006c67;
+  grid-column: 1/ span 3;
+ 
 }
 .sidebar-left {
   background-color: #003844;
+  grid-row: 2 / -1;
+  
+  
 }
 .sidebar-right {
   background-color: #ffb100;
+  
+ }
+
+@media only screen and (max-width: 900px) {
```

Comment:
Commonly used width of small screens is 768px so remove this media query and use the one below instead.